### ISSUE TITLE
some CI fails because of vagrant shared folder configuration

### ISSUE THREAD

**Issue by kenhys** [fluent/fluent-package-builder#593]
# Problem

When checking https://github.com/fluent/fluent-package-builder/pull/592 CI action, it was observed that
testing CI failure.

```
==> debian-bullseye: Matching MAC address for NAT networking...
==> debian-bullseye: Checking if box 'bento/debian-11' version '202309.08.0' is up to date...
==> debian-bullseye: Setting the name of the VM: fluent-package-builder_debian-bullseye_1697186282329_19143
Vagrant is currently configured to create VirtualBox synced folders with
the `SharedFoldersEnableSymlinksCreate` option enabled. If the Vagrant
guest is not trusted, you may want to disable this option. For more
information on this option, please refer to the VirtualBox manual:

  https://www.virtualbox.org/manual/ch04.html#sharedfolders

This option can be disabled globally with an environment variable:

  VAGRANT_DISABLE_VBOXSYMLINKCREATE=1

or on a per folder basis within the Vagrantfile:

  config.vm.synced_folder '/host/path', '/guest/path', SharedFoldersEnableSymlinksCreate: false
==> debian-bullseye: Clearing any previously set network interfaces...
==> debian-bullseye: Preparing network interfaces based on configuration...
    debian-bullseye: Adapter 1: nat
==> debian-bullseye: Forwarding ports...
    debian-bullseye: 22 (guest) => 2222 (host) (adapter 1)
==> debian-bullseye: Running 'pre-boot' VM customizations...
==> debian-bullseye: Booting VM...
==> debian-bullseye: Waiting for machine to boot. This may take a few minutes...
    debian-bullseye: SSH address: 127.0.0.1:2222
    debian-bullseye: SSH username: vagrant
    debian-bullseye: SSH auth method: private key
The guest machine entered an invalid state while waiting for it
to boot. Valid states are 'starting, running'. The machine is in the
'gurumeditation' state. Please verify everything is configured
properly and try again.
```

# Expected

Above CI failure will be solved.

**Comment by kenhys** [comment]
At first, it seems that shared folder configuration is related, but it is not.(it is just example to disable it)
At least, it can't be reproduced on:

```
VBoxManage -v
7.0.10_Debianr158379
vagrant version
Installed Version: 2.3.4
```

It may be better to try on:

https://github.com/actions/runner-images/blob/main/images/macos/macos-12-Readme.md

**Comment by kenhys** [comment]
```
==> debian-bullseye: Successfully added box 'bento/debian-11' (v202309.08.0) for 'virtualbox'!
```

```
==> debian-bookworm: Checking if box 'bento/debian-12' version '202309.08.0' is up to date...
```

It seems that both of image can not be booted with Vagrant 2.3.7, VirtualBox 6.1.38r153438 (See macos-12-Readme.md)

**Comment by kenhys** [comment]
Fixed via #603 

Now migrated to lxc.
